src/config-hash/index.ts

```diff
@@ -0,0 +1,25 @@
+import { createHash } from 'node:crypto';
+
+import { isObject } from 'lodash';
```

Comment:
Same comment as https://github.com/api3dao/commons/pull/54#discussion_r1505004128

```suggestion
import isObject from 'lodash/isObject';
```